### ISSUE TITLE
Using sync storage for Recently Watched List

### ISSUE THREAD

**Issue by lap00zza** [lap00zza/9anime-Companion#82]
**Select your issue type:** (check at least one)
- [ ] Bug
- [ ] Question
- [x] Suggestion
- [ ] Other (please describe):

**Describe your issue:**

**Comment by kr3d** [comment]
1. please extend the amount of anime in the recently watched list, make it so the person can add a cap themselves, so that if they watching 15 anime at the time but just in case they don't want everything to be crowded they add a cap of 25.

**Comment by kr3d** [comment]
2. make it so that the recently watched can be synced to the account.

**Comment by kr3d** [comment]
3. can u make it so people can hide the things on the companion, like the icon that links you to my anime list, link to the 9anime website.

**Comment by kr3d** [comment]
4. could you also make it that in the recently watched, if a new episode comes out next to the anime it says the words [NEW] next to it,

**Comment by kr3d** [comment]
5. this is more to see how it would look like but could you post a screenshot of how the extension would like like in the colours of the 9anime website and ask people to vote weather or not it should be changed.

**Comment by lap00zza** [comment]
@kr3d hey thanks for the suggestions 😄 . Just one small thing. Can you please put your suggestions to a single comment? You can edit comments by pressing the :pen: icon on top right. So something like

```
1. ...
2. ...
3. ...
```
would be perfect. Or if you want I can edit it for you. Lemme know.

**Comment by densityx** [comment]
@lap00zza This is what we have initially planned, a dedicated page for recently watched list

Should we implement it?

**Comment by lap00zza** [comment]
@densityx yeah eventually.  For now what do you think will be better? I was thinking of simply increasing the cap to 20-25.
